openedx/core/djangoapps/user_api/management/commands/populate_enterprise_learner_current_job.py

```diff
@@ -0,0 +1,150 @@
+""" Management command for populaing current job of learners in learner profile. """
+
+import logging
+
+from edx_rest_api_client.client import OAuthAPIClient
+from urllib.parse import urljoin
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from common.djangoapps.student.models import User, UserProfile
+from openedx.core.djangoapps.user_api import errors
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Command to populate current job of learners in extended learner profile.
+
+    This command will fetch the current job of learners from course-discovery service
+    and populate it in the extended learner profile in edx-platform. This command is
+    supposed to be run only once in the system.
+
+    Example usage:
+        $ # Update the current job of learners.
+        $ ./manage.py lms populate_enterprise_learner_current_job
+    """
+
+    help = 'Populates current job of learners in extended learner profile.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--oauth-host', required=False,
+                            default=settings.ENTERPRISE_BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL)
+        parser.add_argument('--client-id', required=False,
+                            default=settings.ENTERPRISE_BACKEND_SERVICE_EDX_OAUTH2_KEY)
+        parser.add_argument('--client-secret', required=False,
+                            default=settings.ENTERPRISE_BACKEND_SERVICE_EDX_OAUTH2_SECRET)
```

Comment:
I am not sure why we need all the above arguments? I think these values will be automatically fetched from stored configuration.

Reply:
Removed these arguments.